graf2d/graf/CMakeLists.txt

```diff
@@ -101,3 +99,5 @@ ROOT_STANDARD_LIBRARY_PACKAGE(Graf
   BUILTINS
     FREETYPE
 )
+
+target_include_directories(Graf PRIVATE ${FREETYPE_INCLUDE_DIRS})
```

Comment:
These sort of changes are better in a separate PR because they are relevant even without the proposed feature while making this PR smaller thus easier to review.

Reply:
They are connected to redesign of build system. Without this PR it was working properly - while $ROOTSYS/include was fine.

Reply:
I disagree with that. In many cases, previously while all was working, we still did these sort of changes because we know it is for better. In a few places we did not, and that should be fixed. That move had nothing to do with this feature, it just made things more consistent and this is what I am asking for those changes, too.